src/filetypes.c

```diff
@@ -951,7 +951,7 @@ static void load_settings(guint ft_id, GKeyFile *config, GKeyFile *configh)
 	{
 		ft->lang = tm_source_file_get_named_lang(result);
 		if (ft->lang < 0)
-			geany_debug("Cannot find tag parser '%s' for custom filetype '%s'.", result, ft->name);
+			geany_debug("Cannot find tags parser '%s' for custom filetype '%s'.", result, ft->name);
```

Comment:
@elextr We are both talking about something different - the phrase "symbols parser" is a nonsense - there's nothing that parses symbols - the output are symbols.

Once again - the settings name for which this message appears is "tag_parser" - do you suggest to change it? It would make the settings file backwards incompatible which isn't a good idea IMO.